### ISSUE TITLE
Notify avatar doesn't show as 1:1 aspect ratio for landscape images

### ISSUE THREAD

**Issue by mesqueeb** [quasarframework/quasar#7959]
**Describe the bug**
Notify avatar doesn't show as 1:1 aspect ratio for landscape images

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/mesqueeb/pen/BazWXQg?editable=true&editors=101%3Dhttps%3A%2F%2Fquasar.dev%2Fquasar-plugins%2Fnotify

<img width="223" alt="image" src="https://user-images.githubusercontent.com/3253920/97258390-ad172d80-185b-11eb-98e8-26ff60d55a7e.png">

**To Reproduce**
1. Go to codepen
2. Click on show notification

**Expected behavior**
Either:
(A) The icon is shown as 1:1 aspect ratio (contain)
or
(B) There is a way to not show this image as weirdly shaped "oval"

**Additional context**
I think it's difficult for all platforms to prepare avatar Urls to square shaped profile pictures. So having some way to accept landscape profile images seems ideal!

**Comment by mesqueeb** [comment]
current workaround, you need this CSS:
```sass
.q-avatar
        height: 1em
        width: 1em
        margin-right: 0.5em
        padding: 0
        > *
            position: relative
            > *
                position: absolute
                top: 0
                width: unset !important
```

**Comment by metalsadman** [comment]
@mesqueeb wrapping the avatar in a div seem to fix it. do you have time to pr on this?

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v1.14.4.
